app/client/ui/GristTooltips.ts

```diff
@@ -187,6 +188,15 @@ see or edit which parts of your document.')
     ),
     ...args,
   ),
+  documentType: (...args: DomElementArg[]) => cssTooltipContent(
+    dom('div',
+      t('Change document type can be useful to transform it into a template or a tutorial')
```

Comment:
@dsagal If you have a better idea for the wording inside the tooltip, it's much than welcome.

Reply:
Was this in the design? I am not sure it's needed, because this is explained in the dialog. But if I missed something, how about "You can turn a document into a template or a tutorial"?

By the way, the preview isn't opening. It looks like there is some typescript error during build. Maybe because the preview (iirc) merges PR with the main branch. So you may want to rebase or merge and check that it builds.

Reply:
Maybe it's a better idea to add it in the same time than the Documentation as it points to nothing at this point

Reply:
There is already a link in the design for "Template" type to "fiddle mode", which explains what this is. The only missing thing is documentation for tutorials, but that's a whole separate documentation effort, because making a tutorial involves other work that's undocumented and unpolished. I think we agreed we are OK to have this feature (to set document type) without waiting for better documentation.

To me, this is enough. I don't even think we should mention creation of templates/tutorials in the "Create document" article, because that article is for new users, and creation of templates/tutorials is only relevant to a small subset of advanced users.

Reply:
Ok, so I remove It.